etcdutl/etcdutl/backup_command.go

```diff
@@ -61,7 +60,6 @@ func NewBackupCommand() *cobra.Command {
 	cmd.Flags().StringVar(&walDir, "wal-dir", "", "Path to the etcd wal dir")
 	cmd.Flags().StringVar(&backupDir, "backup-dir", "", "Path to the backup dir")
 	cmd.Flags().StringVar(&backupWalDir, "backup-wal-dir", "", "Path to the backup wal dir")
-	cmd.Flags().BoolVar(&withV3, "with-v3", true, "Backup v3 backend data")
```

Comment:
Maybe we should preserve the flag and if its set to false print: 
   `-with-v3=false is not supported since etcd v3.6. Please use v3.5.x client as the last supporting this deprecated functionality.`

Reply:
Makes sense.  Resolved & thanks.